tests/unit/multipleBackend/BackendInfo.js

```diff
@@ -7,26 +7,32 @@ const log = new DummyRequestLogger();
 const dummyBackendInfo = new BackendInfo('mem', 'file', '127.0.0.1');
 
 describe('BackendInfo class', () => {
-    describe('isValidControllingBackendParam', () => {
+    describe('controllingBackendParam', () => {
         it('should return false if objectLocationConstraint is invalid', () => {
-            const res = BackendInfo.isValidControllingBackendParam(
+            const res = BackendInfo.controllingBackendParam(
                 'notValid', 'file', '127.0.0.1', log);
-            assert.equal(res, false);
+            assert.equal(res.isValid, false);
+            assert((res.description).indexOf('Object Location Error')
+            > -1);
         });
         it('should return false if bucketLocationConstraint is invalid', () => {
```

Comment:
same idea for this one and the ones below since the function no longer returns false/true